controllers/actions.summerwind.net/autoscaling_test.go

```diff
@@ -61,30 +61,38 @@ func TestDetermineDesiredReplicas_RepositoryRunner(t *testing.T) {
 		want         int
 		err          string
 	}{
+		// case_0
 		// Legacy functionality
-		// 3 demanded, max at 3
+		// 2 demanded, max at 3
```

Comment:
```suggestion
		// 0 demanded due to zero runID, min at 2
```

Perhaps this is better to say!
`want` becomes 2 due to the min replicas set to 2, not due to the workflow runs statuses demanded 2 replicas.